production.env

```diff
@@ -0,0 +1,21 @@
+SECRET_KEY_BASE=""
+PAPERCLIP_HASH_SECRET=""
+
+DATABASE_URL=postgresql://postgres@db/scinote_production
+
```

Comment:
I guess I would rather have a solution where a user needs to run a command to generate a "default" version of this file (which is `.gitignored`), and then update it according to his/her needs. But I don't know if that's easily achievable. 

From maintenance point of view, it would be better than always modifying a file that's tracked in Git.